nesta/core/routines/health_data/nih_data/mesh_root_task.py

```diff
@@ -0,0 +1,22 @@
+import luigi
```

Comment:
Docstring describing what this task chain does (probably being hypocritical here)